pypeit/reduce.py

```diff
@@ -608,7 +618,7 @@ def find_objects_pypeline(self, image, ivar, std=False, std_trace = None, masksl
 
     # JFH TODO Should we reduce the number of iterations for standards or near-IR redux where the noise model is not
     # being updated?
-    def local_skysub_extract(self, sciimg, sciivar, tilts, waveimg, global_sky, rn2img, sobjs,
+    def local_skysub_extract(self, tilts, waveimg, global_sky, rn2img, sobjs,
```

Comment:
Please restore the original argument lists.

Reply:
see above